Source/Core/DolphinQt/GameList/GameTracker.h

```diff
@@ -90,6 +92,7 @@ class GameTracker final : public QFileSystemWatcher
   Common::Event m_initial_games_emitted_event;
   bool m_initial_games_emitted = false;
   bool m_started = false;
+  u32 m_busy_count = 0;
```

Comment:
It may help to have a tiny documentation comment for this variable, given it's not obvious what it's used for just by reading it in isolation (and can prevent it being used incorrectly if code is ever changed in the future).

Reply:
Should this perhaps even be an `atomic`? Or is this considered safe since only one thread (which is `m_load_thread`) should be changing it at any given time?

Reply:
> Or is this considered safe since only one thread (which is `m_load_thread`) should be changing it at any given time?

That is the case indeed, I ensured this only gets accessed from the load thread.